scripts/doginfo.js

```diff
@@ -0,0 +1,115 @@
+  },
+  {
+    h3: 'How To Walk Multiple Dogs',
+    p: 'Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat. <br/><br/>Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.'
+  },
+  {
+    h3: 'Teach Your Dog To Fetch!',
+    p: 'Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat. <br/><br/>Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.'
+  }
+];
+
+const img_container = document.getElementsByClassName('blog-img');
+
+for (let i = 0; i < img_container.length; i += 1){
+  const img_container_item = img_container[i];
+  const images_item = images[i];
+
+  const image = document.createElement('IMG');
+  image.src = images_item.src;
+  image.alt = images_item.alt;
+  img_container_item.appendChild(image);
+};
+
+const blogtext_container = document.getElementsByClassName('blog-textbox');
+
+for (let i = 0; i < blogtext_container.length; i += 1){
+  const blogtext_container_item = blogtext_container[i];
+  const blogtexts_item = blogtexts[i];
+
+  const header3 = document.createElement('h3');
+  header3.innerHTML = blogtexts_item.h3;
+  const paragraph = document.createElement('p');
+  paragraph.innerHTML = blogtexts_item.p;
+  paragraph.setAttribute('class', 'blog-text');
+
+  blogtext_container_item.appendChild(header3);
+  blogtext_container_item.appendChild(paragraph);
+}
+
+//form page//
+$('form').submit(function(e){
+  var thanks = 'Thank you. The form information has been received.';
```

Comment:
include e.preventDefault() at the beginning of the file so the page doesn't refresh. right now you have it in your validation if statement, but it is not being called unless someone is missing a value.

Reply:
I did not included e.preventDefault() after form successful submission, as it would leave the user's information after receiving a confirmation and that's not a typical UX. I did not realize we were expected to leave the form data after successful submission.

Reply:
Update on this: what Jeremy meant was that upon refresh, the data is appended to the website URL which is generally a bad practice when working with sensitive data. This is as easy of a fix as changing for your form from 'get' to 'post'. Also updated your grade. Sorry for the miscommunication on our end.

Reply:
Ah, I see! Thank you for this insight and no worries on the miscommunication at all. I've learned so much in this course and I'm looking forward to finding opportunities to apply the skills.